content/api/ab-testing.apib

```diff
@@ -4,234 +4,239 @@ description: A/B Testing of templates.
-| start_time | string | ISO Date specifying when the test should begin | |
-| end_time | string | (Optional) ISO Date specifying when the test should end | Defaults to (30 days from start_time - engagement timeout) |
-| total_sample_size | int | (Optional) Total number of messages to send as part of the test | |
-| confidence_level | float | (Optional) Specify a confidence level at which point the test should end | Defaults to 0.95 |
-| engagement_timeout | int | (Optional) The amount of time, in hours, until the lack of an engagement event is counted as a failure. | Defaults to 24 hours |
-| created_at | string | ISO Date of A/B Test Creation | GET only |
-| updated_at | string | ISO Date of the last time the A/B test was updated | GET only |
-
-### Template Properties
-| Property   | Type    | Description | Notes |
-|------------|---------|-------------|-------|
-| template_id | string | The template id | |
-| sample_size | integer | The number of injections to send using this template | Required if A/B test has `total_sample_size` defined |
-| percent | integer | The percent of injections to send using this template | Required if A/B test does not have `total_sample_size` defined |
-
-<div class="alert alert-info"><strong>Note</strong>: Tests will run until one of the following criteria are met: The end_time has passed, messages equal to the total_sample_size have been sent, or the confidence_level (Bayesian mode only) has been reached. In Bayesian mode, reaching the specified confidence_level for a template will cause it to become the "winner". If a test ends and the confidence_level has not been reached, the default template will be considered the "winner". </div>
-
-## A/B Tests [/ab-test]
-
-## Create an A/B Test [POST]
-
-+ Request A/B Test using a percentage for distribution
-
-  + Headers
-
-            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
-            Accept: application/json
-
-  + Body
-    ```json
-        {
-          "id": "payment-confirmation",
-          "name": "Payment Confirmation",
-          "metric": "count_unique_confirmed_opened",
-          "audience_selection": "percent",
-          "start_time": "2018-04-03T22:08:33Z",
-          "test_mode": "bayesian",
-          "confidence_level": 0.99,
-          "default_template": {
-            "template_id": "default_payment_confirmation_template",
-            "percent": 50
-          },
-          "variants": [
+An A/B test is a method of comparing templates to see which preforms best. Each A/B test has a default template and up to 20 template variants to compare. You can configure whether we compare the templates based on the number of unique opens or unique clicks.
+
+You can also choose how many messages use each variant in one of two ways:
+* A percentage of messages
+* A fixed number of messages
+
+The test will then run until one of the following criteria is met:
+* The `end_time` passes
+* The number of messages sent equals `total_sample_size`
```

Comment:
I wonder if we should make the language around sample size a little loose.  We just run metrics queries at fixed intervals to figure out how much they've sent, and stop if they've exceeded the threshold, so it's certainly not exact.

Also we base things on count accepted, not injected.  So maybe this should read something like "The number of messages accepted equals or exceeds `total_sample_size`"